Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    yext-api (0.1.3)
+    yext-api (0.1.5)
```

Comment:
I think this will likely need changing since I had 2 PRs opened when this was created.